java/map-image-layer-tables/src/main/res/layout/service_comments.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:background="#FFFFFF">
+
+    <ListView android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:id="@+id/comment_list"
+              android:divider="#7F7F7F"
+              android:dividerHeight="1dp"/>
```

Comment:
maybe add 'android:fadeScrollbars="false"' it wasnt' clear to me at first that the top window was a listview